chainercv/extensions/__init__.py

```diff
@@ -1 +1,2 @@
 from chainercv.extensions.detection.detection_vis_report import DetectionVisReport  # NOQA
+from chainercv.extensions.detection.detection_voc_evaluator import DetectionVOCEvaluator  # NOQA
```

Comment:
But, I think that `*<TYPE_NAME>` is a ubiquitous rule. (e.g. TransformDataset is a dataset class)
Do you think it is worth it to break this rule?